graph/metafeeds_test.go

```diff
@@ -83,6 +85,134 @@ var metafeedsScenarios = []PeopleTestCase{
 			PeopleAssertHops("alice", 0, "alice-legacy", "some"),
 		},
 	},
+
+		// name: "main announces fake metafeed, should not work",
+		// name: "followed peer has newly announced metafeed",
+	{
+		name: "follow peer with metafeed announcement",
+		ops: []PeopleOp{
+			PeopleOpNewPeer{"alice-main"},
+			PeopleOpNewPeer{"bob-main"},
+			PeopleOpFollow{"alice-main", "bob-main"},
+			PeopleOpNewPeerWithAlgo{"bob-mf", refs.RefAlgoFeedBendyButt},
+			PeopleOpAnnounceMetafeed{"bob-main", "bob-mf"},
+			// question: do we need to add a new op for adding an existing subfeed?
+			// i.e. PeopleOpExistingSubFeed, instead of PeopleOpNewSubFeed
+
+			// question: how do we actually test that the announcement works correctly?
+			// i assume we need to create a normal feed (normie) following bob-main, and then check that 
+			// the hops from normie->{bob-mf,bob-main,bob-indexes: about} == 0 (using hops 0 according to go-ssb hops conventions)
+			//
+			// following that logic: we should also have a test that maliciously creates an announcement for a metafeed that
```

Comment:
I'm fairly certain we don't need this.

TODO: make this more clear in the the metafeed spec?